dymos/run_problem.py

```diff
@@ -83,3 +83,5 @@ def run_problem(problem, refine_method='hp', refine_iteration_limit=0, run_drive
         else:
             timeseries_plots(solution_record_file, simulation_record_file=None,
                              plot_dir=plot_dir)
+            
```

Comment:
Hello and thanks for your contribution!  This one is ready to be merged except for failing our formatting test due to whitespace on this otherwise blank line.

If you can fix that and re-push, we'll get it merged in.  If you want, I can take care of this for you.

Reply:
Hello and thank you for the feedback. I deleted the whitespaces.